### PR TITLE
update the handling of lp to the current parmed version

### PR DESCRIPTION
**Pull request by JohannesKarwou**
parmed now supports lonepairs and we are no longer relying on our own forked version. If LPs are part of the dummy region electrostatics are turned off together with the other dummy atoms. More is not necessary since they do not have any LJ interactions and can be never the terminal dummy atom since they are always connected to a chlorine or bromine atom.

### CODE REVIEW

File: transformato/annihilation.py [hunk 1]
```diff
@@ -325,6 +333,11 @@ def _mutate_to_common_core(
                 if atom_idx in hydrogens:
                     # is already mutated in the previouse step
                     continue
+                elif self.psf1["waterbox"].atoms[atom_idx].type == "LPH":
```
Comment (JohannesKarwou) [inline transformato/annihilation.py:336]: LP should not have  lj interaction, so they do not need to be turned off in one seperate step. **ATTENTION: `self.psf1` needs to be changed to be flexible for RBFE**

**Comment by JohannesKarwou** [comment]
I added this pull request (https://github.com/wiederm/transformato/pull/90) into the current one to get one working branch.
Pull request https://github.com/wiederm/transformato/pull/90 is about integrating the `ProposeMutationRouteASFE` from the `annihilation.py` into the `ProposeMutationRoute` from the `mutation.py` file

File: transformato/bin/perform_correction.py [hunk 2]
```diff
@@ -79,6 +80,10 @@
     )
 
     r = perform_endstate_correction(fep_protocoll)
+    
+    with open(f"results_{system_name}.pickle", "wb") as file:
+        pickle.dump(r, file)
+
```
Comment (wiederm) [inline transformato/bin/perform_correction.py:86]: I am never happy to write out pickle files if we can also use csv.

File: transformato/tests/test_absolute.py [hunk 3]
```diff
@@ -134,25 +133,36 @@ def test_compare_mda_and_mdtraj():
...
+    os.getenv("CI") == "true",
+    reason="Skipping tests that cannot pass in github actions",
+)
+def test_create_asfe_system_with_lp():
+
+    from transformato.mutate import ProposeMutationRoute
 
     configuration = load_config_yaml(
-        config=f"{get_testsystems_dir()}/config/methanol-asfe.yaml",
+        config=f"{get_testsystems_dir()}/config/1,3-dichlorobenzene-asfe.yaml",
```
Comment (wiederm) [inline transformato/tests/test_absolute.py:146]: that seems like a dangerous file name :-)

**Review by wiederm — APPROVED** [review]
LGTM! but a lot have changed, can we talk about these changes in a quick call?

**Pull request merged**